docs/source/getting_started.rst

```diff
@@ -320,35 +320,140 @@ syntax like so:
    # If you want compatibility with even older versions of Python
    my_global_dict = {}  # type: Dict[int, float]
 
-.. _stubs-intro:
+.. _pep526: https://www.python.org/dev/peps/pep-0526/
+
 
-Library stubs and typeshed
-**************************
+Types and classes
+*****************
 
-Mypy uses library *stubs* to type check code interacting with library
-modules, including the Python standard library. A library stub defines
-a skeleton of the public interface of the library, including classes,
-variables and functions, and their types. Mypy ships with stubs for
-the standard library from the `typeshed
-<https://github.com/python/typeshed>`_ project, which contains library
-stubs for the Python builtins, the standard library, and selected
-third-party packages.
+So far, we've only seen examples of pre-existing types like the ``int``
+or ``float`` builtins, or the ``List`` and ``Dict`` types from the typing
+module. However, these aren't the only types you can use: in fact, you can
```

Comment:
This is now out of date; the previous section has been updated to use PEP 585 syntax.

```suggestion
or ``float`` builtins, or generic types from ``collections.abc`` and
``typing``, such as ``Iterable``. However, these aren't the only types you can
use: in fact, you can
```